packages/babel-plugin-transform-function-name/test/fixtures/function-name/unicode-encoding/options.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You can delete this test, since we already test the explicit "is supported" and "is not supported" cases. This one is causing CI to fail because it is in the "is supported" category for Babel 8 and in the "is not supported" category for Babel 7.

Reply:
Okay, I will remove the following test.